proto/one/operations/sample/analyte.proto

```diff
@@ -0,0 +1,35 @@
+syntax = "proto3";
+import "google/protobuf/wrappers.proto";
+import "record_auditinfo.proto";
+option csharp_namespace ="ONE.Models.CSharp";
+
+message Analyte {
+
+    // Unique identifier of the analyte
+    string id = 1;
+    
+    // Used to check twin authorization
+    google.protobuf.StringValue authTwinRefId = 2;
```

Comment:
@ChrisMarin-AI  / @joezhou-aqi  - every google.protobuf... property will be nullable.  It seems like almost every field is nullable for these classes.  Was that by design?

Reply:
I will make the changes for the non nullable fields.

Reply:
@ChrisMarin-AI , please check documentation for those properties and make appropriate changes accordingly.